### ISSUE TITLE
Deciding on Version Scheme and How we are going to do different blender versions. 

### ISSUE THREAD

**Issue by Yusarina** [unofficalcats/Avatar-Toolkit#3]
I though I add this here so it doesn't get lost in Discord.

We need to decide on how we are going to do versions in the Avatar Toolkit. Currently in Cats we do each version such as 3.6.x.x for Blender 3.6, 4.1.x.x for blender 4.1 and so on. Now this was easy when bl_info was a thing as it allowed us to do this version scheme. But with the new extension system manifest it only allows us to do 3 numbers like 3.6.1. 

Now as an example I guess we could do something like 5.0.12, 5.0.15 for blender 5.0 or 5.1.12, 5.1.15 for Blender 5.1. 

Also I want to know do we support older version or do we roll on a system that we just support the latest LTS and if older versions break, there break, what do you think?

**Comment by 989onan** [comment]
I was thinking that having version numbers like blender's 5.1 would be good, and then hold off new features for new blender versions, to make it so that it matches blender's schema of adding new features with major versions

**Comment by Yusarina** [comment]
Ok, so we just do 1.0 for the first stable release then 1.1 for a minor release if this is what you mean?

My only issue with this that it's harder to tell which version is for what blender version so we would need to clearly mark it for what version of blender.

What I don't want to try and do is try and support older versions of blender, like Cats was trying to do. We should only support the latest LTS and latest version, like I basically doing for Cats right now. Do you agree?

**Comment by 989onan** [comment]
Yes this seems like a good idea, and just tell people what version the plugin currently supports in the read me
And if we do release packages in releases, then specifying there what version it is for when people want an older version of the plugin for god knows what reason, then they can get an outdated version of the plugin for their blender version.

**Comment by Yusarina** [comment]
Ok to just make this clear, this is what we want.

- Version numbers will be like 1.0, 1.1 and etc.
- We are only going to support the latest LTS and the latest Blender version. For example we would support Blender 4.2 LTS and then Blender 4.3. But when Blender 5.0 is released we would no longer support Blender 4.3. But now Blender 4.2 LTS and Blender 5.0. The same if a new LTS is released. 
- On packages both in the title we would put in brackets the version number of Toolkit and the blender versions for example 1.1 (Blender 5.0 and 5.1). We should also add this to the description.
- We will only add new features to new blender version releases, only bug fixes and etc for older version we support. 

This sound about right?

**Comment by 989onan** [comment]
this sounds perfect, let us do that

**Comment by Yusarina** [comment]
Awesome, we keep this open mainly so I can remind myself, but I write up something proper at somepoint.
